SoloProjects/clc80/Dashboard/Dashboard/DashboardCard.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  DashboardCard.swift
+//  Dashboard
+//
+//  Created by Claudia Maciel on 12/13/20.
+//
+
+import UIKit
+
+class DashboardCard: UIView {
+    //MARK: - Properties
+    var image1 = UIImageView()
+    var image2 = UIImageView()
+    let title = UILabel()
+    let subTitle = UILabel()
+    
+    override init(frame: CGRect) {
+        super .init(frame: frame)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    init(image1: UIImageView, image2: UIImageView, title: String, subTitle: String) {
+        super .init(frame: .zero)
+        self.image1 = image1
+        self.image2 = image2
+        self.title.text = title
+        self .subTitle.text = subTitle
+        configure()
+    }
+    
+    private func configure() {
+        layer.cornerRadius = 10
+        self.backgroundColor = .systemPink
+        
+        self.addSubview(image1)
+        self.addSubview(image2)
+        self.addSubview(title)
+        self.addSubview(subTitle)
+        
+        translatesAutoresizingMaskIntoConstraints = false
+        image1.translatesAutoresizingMaskIntoConstraints = false
+        image2.translatesAutoresizingMaskIntoConstraints = false
+        title.translatesAutoresizingMaskIntoConstraints = false
+        subTitle.translatesAutoresizingMaskIntoConstraints = false
+     
+        title.font = UIFont.preferredFont(forTextStyle: .title1)
+        subTitle.font = UIFont.preferredFont(forTextStyle: .subheadline)
+        subTitle.textColor = .gray
+        
+        NSLayoutConstraint.activate([
+            image1.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 20),
+            image1.topAnchor.constraint(equalTo: self.topAnchor, constant: 20),
+            image1.heightAnchor.constraint(equalToConstant: 50),
+            image1.widthAnchor.constraint(equalToConstant: 50),
+            
+            title.leadingAnchor.constraint(equalTo: image1.leadingAnchor),
+            title.topAnchor.constraint(equalTo: image1.bottomAnchor, constant: 20),
+            
+            image2.bottomAnchor.constraint(equalTo: title.centerYAnchor),
+            image2.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -50),
+            image2.widthAnchor.constraint(equalToConstant: 75),
+            image2.heightAnchor.constraint(equalToConstant: 50),
+            
+            subTitle.leadingAnchor.constraint(equalTo: image1.leadingAnchor),
+            subTitle.topAnchor.constraint(equalTo: title.bottomAnchor, constant: 8),
```

Comment:
The reason you're not seeing a background color is because the DashboardCard has an ambiguous height, given the constraints you have here. I opened your app in Xcode's visual debugger to help diagnose this:

<img width="1440" alt="Screen Shot 2021-01-10 at 2 56 51 PM" src="https://user-images.githubusercontent.com/720600/104135214-51d1e300-5354-11eb-96cc-c10a8c7c50e7.png">

UIKit can't determine what the height is supposed to be, so it's using `0`, which is the height it's initialized with. The reason the height is ambiguous is because your constraints don't have a contiguous chain of constraints from the top of the card to the bottom of the card. Right now the chain is broken:

- START: top of the card
- top of the card to top of image 1
- bottom of image1 to top of title label
- bottom of title label to top of subtitle label
- BROKEN
- END: bottom of the card

You need a constraint between the bottom of the subtitle label and the bottom of the card.